dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/DubboProtocol.java

```diff
@@ -384,31 +379,37 @@ private void optimizeSerialization(URL url) throws RpcException {
     /**
      * Get shared connection
      */
-    private ExchangeClient getSharedClient(URL url) {
+    private List<ReferenceCountExchangeClient> getSharedClient(URL url) {
         String key = url.getAddress();
-        ReferenceCountExchangeClient client = referenceClientMap.get(key);
-        if (client != null) {
-            if (!client.isClosed()) {
-                client.incrementAndGetCount();
-                return client;
-            } else {
-                referenceClientMap.remove(key);
-            }
+        List<ReferenceCountExchangeClient> clients = referenceClientMap.get(key);
+        if(clients == null) {
+            List<ReferenceCountExchangeClient> referenceCountExchangeClients = buildReferenceCountExchangeClientList(url, key, connectNum);
```

Comment:
ReferenceCountExchangeClient is for sharing connections. I saw your pr. Your pr wants to create multiple connections instead of sharing connections by default. In this case, what is the purpose of using ReferenceCountExchangeClient?

Reply:
It's reasonable. I'll optimize it again.

Reply:
Hello, if you just change the default share to multiple, then just change the default value of connections? How do u think about it?
:)

Reply:
Is it convenient to leave a contact? My phone number: 18668225833